### ISSUE TITLE
Incompatible with KeePass auto-delete

### ISSUE THREAD

**Issue by Hexaae** [wiziple/1Clipboard#17]
For obvious security reasons KeePass has an option to auto-delete last clipboard, after you copied it by double-clicking. It works with ClipX, CLCL and the rest, except 1Clipboard.

**Comment by noformnocontent** [comment]
just add your passwords to `.clipignore`

**Comment by LoganDark** [comment]
Yeah, store all your passwords easily accessible in a plaintext file.

**Comment by noformnocontent** [comment]
just fyi, `/s` in my comment is short for `</sarcasm>`

**Comment by LoganDark** [comment]
Oh.

**Comment by Hexaae** [comment]
I see no activity from devs... just 1 closed issue in many months

**Comment by noformnocontent** [comment]
about the state of the development:

> I am currently working on this project from scratch with the newer web technology, so the source code is different from the released version. I am going to open this project as a public repository in the future when it is ready.

plus regular commits to some private and several `react` themed repos, so expect it soon™

----

also, for windows i would suggest [Ditto](http://ditto-cp.sourceforge.net)
